frontend/src/interfaces/category.ts

```diff
@@ -1,4 +1,5 @@
 export default interface Category {
+  id?: number,
```

Comment:
`id` is nullable?

Reply:
Sa database kay dili sir, auto increment ang `id`.